### PR TITLE
Add invoice retry autopay instruction

### PR DESCRIPTION
**Pull request by adamaltman**

### CODE REVIEW

File: openapi/components/schemas/InvoiceRetry/InvoiceRetryInstruction.yaml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+properties:
+  attempts:
+    type: array
+    minItems: 1
+    items:
+      type: object
+      properties:
+        scheduleInstruction:
+          $ref: ../Scheduling/InvoiceRetryScheduleInstruction.yaml
+        autopayInstruction:
```
Comment (VladimirBerdnik) [inline openapi/components/schemas/InvoiceRetry/InvoiceRetryInstruction.yaml:12]: I wonder if this parameter should be named `amountAdjustmentInstruction` so we can read it like `An attempt's amount adjustment instruction` and not `An attempt's autopay instruction`

Comment (adamaltman) [reply openapi/components/schemas/InvoiceRetry/InvoiceRetryInstruction.yaml:12]: Please go ahead and make that change and commit it to this branch.

File: openapi/components/schemas/InvoiceRetry/InvoiceRetryAmountAdjustmentInstruction.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+type: object
+discriminator:
+  propertyName: method
+  mapping:
+    partial: ./Policies/partial.yaml
+    none: ./Policies/None.yaml
```
Comment (VladimirBerdnik) [inline openapi/components/schemas/InvoiceRetry/InvoiceRetryAmountAdjustmentInstruction.yaml:6]: @adamaltman , this `None` seems of another scope - after approval policies, not amount adjustment policies. I think I can separate them by different directories to increase readability if you don't mind. Then we can add 'none' for amount adjustment policy as well

Comment (adamaltman) [reply openapi/components/schemas/InvoiceRetry/InvoiceRetryAmountAdjustmentInstruction.yaml:6]: Please go ahead. I found this a mess to begin with, and I probably made it even worse. Thank you.

Comment (VladimirBerdnik) [reply openapi/components/schemas/InvoiceRetry/InvoiceRetryAmountAdjustmentInstruction.yaml]: Separated by this commit: https://github.com/Rebilly/api-definitions/pull/36/commits/e1063fb9206b3faa818b5d91d24a9e92ad26ccc5
Fount missed `none` in PaymentRetry as well: https://github.com/Rebilly/api-definitions/pull/36/commits/b5a181ef489899f8ba2f9ebdc0546c099d45ca76

File: openapi/components/schemas/InvoiceRetry/AmountAdjustmentInstructions/Policies/ScheduleAmountRemaining.yaml [hunk 3]
```diff
@@ -0,0 +1,10 @@
+title: Schedule autopay for amount remaining
```
Comment (2e3s) [inline openapi/components/schemas/InvoiceRetry/AmountAdjustmentInstructions/Policies/ScheduleAmountRemaining.yaml:1]: I suppose this one isto be removed for now

Comment (adamaltman) [reply openapi/components/schemas/InvoiceRetry/AmountAdjustmentInstructions/Policies/ScheduleAmountRemaining.yaml:1]: Yes, I propose it is out of scope.

**Review by agualis — DISMISSED** [review]

**Comment by agualis** [comment]
Sorry, I approved this PR by mistake. I think that I cannot undo it so please ignore that approval.

File: openapi/components/schemas/InvoiceRetry/AmountAdjustmentInstructions/Policies/ScheduleAmountRemaining.yaml (hunk 3 above)
Comment (VladimirBerdnik) [reply openapi/components/schemas/InvoiceRetry/AmountAdjustmentInstructions/Policies/ScheduleAmountRemaining.yaml]: The file was removed

File: openapi/components/schemas/InvoiceRetry/AmountAdjustmentInstructions/partial.yaml [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+        description: The payment amount
+        type: number
+        format: float
+      type:
+        description: The payment amount type
+        type: string
+        enum:
+          - percent
+          - fixed
+      afterApprovalPolicies:
```
Comment (2e3s) [inline openapi/components/schemas/InvoiceRetry/AmountAdjustmentInstructions/partial.yaml:15]:
```suggestion
      afterApprovalPolicy:
```

**Review by VladimirBerdnik — DISMISSED** [review]

**Review by 2e3s — DISMISSED** [review]

**Review by iamdevice — DISMISSED** [review]

**Review by VladimirBerdnik — APPROVED** [review]

**Review by iamdevice — APPROVED** [review]

**Pull request merged**